app/code/community/Bolt/Boltpay/OrderCreationException.php

```diff
@@ -0,0 +1,214 @@
+<?php
+/**
+ * Bolt magento plugin
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * http://opensource.org/licenses/osl-3.0.php
+ *
+ * @category   Bolt
+ * @package    Bolt_Boltpay
+ * @copyright  Copyright (c) 2018 Bolt Financial, Inc (https://www.bolt.com)
+ * @license    http://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
+ */
+
+/**
+ * Class Bolt_Boltpay_OrderCreationException
+ *
+ * This exception is thrown when an error occurs while trying to create a backend order
+ */
+class Bolt_Boltpay_OrderCreationException extends Bolt_Boltpay_BoltException
+{
+
+    /**
+     * Database error or error in incoming request
+     */
+    const E_BOLT_GENERAL_ERROR               = 2001001; // verified
```

Comment:
what does `verified` mean?

Reply:
it was a marker that I used when I manually verified all error codes by manipulating the magento admin.  I forgot to remove these markers